.github/workflows/standard-deploy.yml

```diff
@@ -85,6 +106,21 @@ jobs:
         copilot svc init --app pre-award --name fsd-${{ inputs.app_name }}
         copilot svc deploy --env ${{ inputs.environment }} --app pre-award --name fsd-${{ inputs.app_name }}
 
+    - name: Slack message for end of deployment
+      if: ${{ always() && inputs.notify_slack_on_deployment && (steps.slack_start_deployment_message.outcome == 'success' && inputs.environment == 'prod' || inputs.environment == 'production') }}
```

Comment:
Will the `always()` change any behaviour here?

Reply:
nope, it just makes sure it still fires if if the previous steps failed